README.md

```diff
@@ -3,7 +3,7 @@
 # Sentinel: Sentinel of Your Application
 
 [![Travis Build Status](https://travis-ci.org/alibaba/Sentinel.svg?branch=master)](https://travis-ci.org/alibaba/Sentinel)
-[![Maven Central](https://img.shields.io/maven-central/v/com.alibaba.csp/sentinel-core.svg?label=Maven%20Central)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.alibaba.csp%22%20a%3A%22sentinel-core%22)
+[![Maven Central](https://img.shields.io/maven-central/v/com.alibaba.csp/sentinel-core.svg?label=Maven%20Central)](https://search.maven.org/classic/#search%7Cga%7C1%7Cg%3A%22com.alibaba.csp%22%20a%3A%22sentinel-core%22)
```

Comment:
Hi, you may fix with  https://search.maven.org/search?q=g:com.alibaba.csp%20AND%20a:sentinel-core instead of the classic one.

Reply:
Of course,thanks.In fact, I started a project to deal with maven badges.Check it out if you are interested in it.
[https://github.com/Ivan97/maven-badges-generator](https://github.com/Ivan97/maven-badges-generator)